### DISCUSSION TITLE
Documentation So bad

### DISCUSSION THREAD

**Discussion by fzsf163** [pikasojs/pikaso#106] [General]
The documentation in https://pikaso.app is really bad. It does not tell me anything. Any new comer will have hell to fight to understand the lib. It does not work how it says. The canvas is uncontrollable for any newcomer. 
 
Do you guys have anything for newcomers to get started with ? 
How to control the width and height of actual canvas and the overlay ? Difference between bg image and overlay  and the board whatever that is. All the tutorial is for js, and only one hook for react and a little for ts.  Learn more or full documentation just takes here and there and does say nothing. Most of it left for interpretation. 

Is it dead already ? Am I using something that is not maintained anymore ?

**Comment by raminious** [comment]
Hi @fzsf163 
Thank you for providing your valuable feedback.
Kindly provide your insights regarding potential areas for improvement in the documentation. Your assistance in updating document pages is also greatly appreciated, given that this is an open source project.
Moreover, the project is not dead. In the past months, there wasn't a contributions to the library because it's sufficiently stable and I've already implemented a large number of features that the community requested or that I myself desired to include.

**Reply by raminious** [reply]
I suggest you to start with coding examples.

**Reply by fzsf163** [reply]
Well I tried. it takes a lot of effort to understand how to operate with it. I changed my mind. It is good. I will try to update it's documentation as I learn more about it. Thank you for making it. And thanks for the reply.
